duckduckgo/Android
# Bump retrofit and remove retrofitCoroutinesAdapter dependency

Task/Issue URL: https://app.asana.com/0/414730916066338/1199111180919669/f
Tech Design URL: 
CC: 

**Description**:
This PR updates retrofit version and remove the deprecated dependency with `retrofitCoroutinesAdapter`.

**Steps to test this PR**:
1. Fresh install
1. Go to Leave feedback inside settings
1. Send any feedback (postive/negative)
1. Ensure in the logs you see `Successfully submitted feedback`

Disclaimer: we noticed that this API is pretty slow (up to 5 seconds), when testing be aware of this when waiting for the log to appear.

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
